packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -614,10 +616,12 @@ export class TabBarRenderer extends TabBar.Renderer {
                 this.selectionService.selection = NavigatableWidget.is(widget) ? { uri: widget.getResourceUri() } : widget;
             }
 
+            const contextKeyServiceOverly = this.contextKeyService?.createOverlay([['isTerminalTab', widget && '_terminalId' in widget]]);
```

Comment:
nitpick: there is a typo here with a missing 'a' => contextKeyServiceOverl**a**y